package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-can",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
I bumped the version here so people aren't surprised by not running the generator.